app/assets/javascripts/admin/tasktype/task_type_list_view.js

```diff
@@ -182,12 +182,24 @@ class TaskTypeListView extends React.PureComponent<Props, State> {
                 dataIndex="settings"
                 key="allowedModes"
                 width={100}
-                render={settings =>
-                  settings.allowedModes.map(mode => (
-                    <Tag key={mode} color={mode === settings.preferredMode ? "blue" : null}>
-                      {mode}
-                    </Tag>
-                  ))
+                render={(settings, taskType) =>
+                  [
+                    taskType.tracingType === "skeleton" ? (
+                      <Tag color="blue" key="tracingType">
```

Comment:
The blue tag color is the same as for the preferredMode, maybe change the preferredMode one to green or so.